tests/verify.test.ts

```diff
@@ -0,0 +1,40 @@
+import { ProofType } from '../src';
+import { performVerifyTransaction, performVKRegistrationAndVerification, loadProofData, loadVerificationKey } from './common/utils';
+
+jest.setTimeout(300000);
+
+describe('verify and subscribe to zkVerify proof events without waiting for Attestation event', () => {
+    const proofTypes = Object.keys(ProofType).map((key) => ProofType[key as keyof typeof ProofType]);
+
+    test.each(proofTypes)('should verify %s proof and respond on finalization', async (proofType: ProofType) => {
+        console.log(`Running test for proof type: ${proofType}`);
+
+        const { proof, publicSignals } = loadProofData(proofType);
+        const vk = loadVerificationKey(proofType);
+
+        await performVerifyTransaction(proofType, proof, publicSignals, vk, false);
+    });
+});
+
+describe('verify and subscribe to zkVerify events, wait for Attestation event then call proof of existence', () => {
+    const proofTypes = Object.keys(ProofType).map((key) => ProofType[key as keyof typeof ProofType]);
+
+    test.each(proofTypes)('should verify %s proof and respond on finalization with attestation confirmation then check proof of existence', async (proofType: ProofType) => {
+        console.log(`Running test for proof type: ${proofType}`);
+        const { proof, publicSignals } = loadProofData(ProofType.groth16);
+        const vk = loadVerificationKey(ProofType.groth16);
+
+        await performVerifyTransaction(ProofType.groth16, proof, publicSignals, vk, true, true);
```

Comment:
```suggestion
        const { proof, publicSignals } = loadProofData(proofType);
        const vk = loadVerificationKey(proofType);

        await performVerifyTransaction(proofType, proof, publicSignals, vk, true, true);
```

Reply:
Yes, was originally just testing groth1) but needs updating to be all proof types thanks 😃